src/views/Details/Details.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <div class="details-wrapper">
+    <NavBar :showBackButton="true" :backClick="goBack" :backLabel="'Back'">
+      {{ title }}
+    </NavBar>
+    <router-view></router-view>
+  </div>
+</template>
+
+<script>
+import { mapGetters } from 'vuex'
+import NavBar from '@/components/NavBar.vue'
+
+export default {
+  components: {
+    NavBar
+  },
+  data () {
+    return {
+      advanced: false,
```

Comment:
I'm confused as to what's going on here
this data is not used in the template
and why is there a router-view inside? Can't this page just be for it's own path?